src/Enum.php

```diff
@@ -82,20 +90,19 @@ public function getValue()
     /**
      * Returns the enum key (i.e. the constant name).
      *
-     * @psalm-pure
-     * @return mixed
+     * @psalm-mutation-free
      */
-    public function getKey()
+    public function getKey(): string
```

Comment:
I guess this is a BC break and I'll revert it.

Reply:
I'm afraid so unfortunately :/ (I'd love to add this but…)

Reply:
done